Logstream fail

```
panic: runtime error: index out of range [0] with length 0
goroutine 77 [running]:
knative.dev/pkg/test/logstream/v2.(*namespaceSource).handleLine(0xc000467ce0, 0xc00073ccba, 0x7eb, 0x1346, 0xc0003ba2e0, 0x19)
	/home/prow/go/src/knative.dev/serving/vendor/knative.dev/pkg/test/logstream/v2/stream.go:210 +0x7cc
knative.dev/pkg/test/logstream/v2.(*namespaceSource).startForPod.func1(0xc00020b7b0, 0x9, 0xc000467ce0, 0xc000441fb0, 0x24, 0xc0003ba2e0, 0x19, 0xc000b803c0)
	/home/prow/go/src/knative.dev/serving/vendor/knative.dev/pkg/test/logstream/v2/stream.go:146 +0x3a7
created by knative.dev/pkg/test/logstream/v2.(*namespaceSource).startForPod
	/home/prow/go/src/knative.dev/serving/vendor/knative.dev/pkg/test/logstream/v2/stream.go:128 +0x2a6
FAIL	knative.dev/serving/test/e2e/autotls	19.097s

```
https://prow.knative.dev/view/gs/knative-prow/pr-logs/pull/knative_serving/10263/pull-knative-serving-istio-stable-no-mesh-tls/1334639944722812928

/cc @mattmoor @dprotaso

--- a/test/logstream/v2/stream.go
+++ b/test/logstream/v2/stream.go
@@ -172,12 +172,12 @@ const (
 func (s *namespaceSource) handleLine(l []byte, pod string) {
 	// This holds the standard structure of our logs.
 	var line struct {
-		Level      string    `json:"level"`
-		Timestamp  time.Time `json:"ts"`
+		Level      string    `json:"severity"`
+		Timestamp  time.Time `json:"timestamp"`
 		Controller string    `json:"knative.dev/controller"`
 		Caller     string    `json:"caller"`
 		Key        string    `json:"knative.dev/key"`
-		Message    string    `json:"msg"`
+		Message    string    `json:"message"`
 		Error      string    `json:"error"`
 
 		// TODO(mattmoor): Parse out more context.

--- a/test/logstream/v2/stream_test.go
+++ b/test/logstream/v2/stream_test.go
@@ -43,7 +43,7 @@ import (
 const (
 	noLogTimeout = 100 * time.Millisecond
 	testKey      = "horror-movie-2020"
-	testLine     = `{"level":"debug","ts":"2020-10-20T18:42:28.553Z","logger":"controller.revision-controller.knative.dev-serving-pkg-reconciler-revision.Reconciler","caller":"controller/controller.go:397","msg":"Adding to queue default/s2-nhjv6 (depth: 1)","commit":"4411bf3","knative.dev/pod":"controller-f95b977c-4wlh4","knative.dev/controller":"revision-controller","knative.dev/key":"default/horror-movie-2020", "error":"el-otoño-eternal" }`
+	testLine     = `{"severity":"debug","timestamp":"2020-10-20T18:42:28.553Z","logger":"controller.revision-controller.knative.dev-serving-pkg-reconciler-revision.Reconciler","caller":"controller/controller.go:397","message":"Adding to queue default/s2-nhjv6 (depth: 1)","commit":"4411bf3","knative.dev/pod":"controller-f95b977c-4wlh4","knative.dev/controller":"revision-controller","knative.dev/key":"default/horror-movie-2020", "error":"el-otoño-eternal" }`
 )
 
 var pod = &corev1.Pod{

